src/firefly/js/ui/FileUploadProcessor.jsx

```diff
@@ -47,10 +47,10 @@ const imageWarning = 'Only loading the table(s), ignoring any selected image(s).
 const tableWarning = 'Only loading the image(s), ignoring any selected table(s).';
 
 
-export function resultSuccess(request) {
+export async function resultSuccess(request) {
```

Comment:
`async` is added, why?

Reply:
I was testing calling some async functions in JobInfo.jsx from here..removed that code but forgot to remove async from here, fixed in the PR update now